openldap-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,86 @@
+local g = import './g.libsonnet';
+local logslib = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+
+{
+  local root = self,
+  new(this):
+    local prefix = this.config.dashboardNamePrefix;
+    local links = this.grafana.links;
+    local tags = this.config.dashboardTags;
+    local uid = g.util.string.slugify(this.config.uid);
+    local vars = this.grafana.variables;
+    local annotations = this.grafana.annotations;
+    local refresh = this.config.dashboardRefresh;
+    local period = this.config.dashboardPeriod;
+    local timezone = this.config.dashboardTimezone;
+    local panels = this.grafana.panels;
+
+    {
+      overview:
+        g.dashboard.new(prefix + 'OpenLDAP Overview')
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            [
+              panels.uptime { gridPos+: { w: 8 } },
+              panels.referrals { gridPos+: { w: 8 } },
+              panels.alerts { gridPos+: { w: 8 } },
+              panels.connections { gridPos+: { w: 8 } },
+              panels.waiters { gridPos+: { w: 8 } },
+              panels.directoryEntries { gridPos+: { w: 8 } },
+              panels.networkConnectivity { gridPos+: { w: 8 } },
+              panels.pduProcessed { gridPos+: { w: 8 } },
+              panels.authenticationAttempts { gridPos+: { w: 8 } },
+              panels.coreOperations { gridPos+: { w: 12 } },
+              panels.auxiliaryOperations { gridPos+: { w: 12 } },
+              panels.primaryThreadActivity { gridPos+: { w: 12 } },
+              panels.threadQueueManagement { gridPos+: { w: 12 } },
+            ], 12, 6
+          )
+        )
+        + root.applyCommon(vars.multiInstance, uid + '-overview', tags, links, annotations, timezone, refresh, period),
```

Comment:
Can you add the option within the links here to hide the link on the dashboard you are in. Pgbouncer and istio has an example of this.